pybv/io.py

```diff
@@ -246,43 +245,41 @@ def _optimize_channel_unit(resolution, unit):
 
 
 def _write_vhdr_file(vhdr_fname, vmrk_fname, eeg_fname, data, sfreq, ch_names,
-                     orientation='multiplexed', format='binary_float32',
-                     resolution=1e-7, unit='µV'):
+                     orientation, format, resolution, unit):
     """Write BrainvVision header file."""
-    fmt = format.lower()
     bvfmt, _ = _chk_fmt(format)
 
     multiplexed = _chk_multiplexed(orientation)
 
     with codecs.open(vhdr_fname, 'w', encoding='utf-8') as fout:
         print(r'Brain Vision Data Exchange Header File Version 1.0', file=fout)
-        print(r';Written using pybv {}'.format(__version__), file=fout)
+        print(r'; Written using pybv {}'.format(__version__), file=fout)
         print(r'', file=fout)
         print(r'[Common Infos]', file=fout)
         print(r'Codepage=UTF-8', file=fout)
         print(r'DataFile={}'.format(op.basename(eeg_fname)), file=fout)
         print(r'MarkerFile={}'.format(op.basename(vmrk_fname)), file=fout)
 
-        if fmt.startswith('binary'):
+        if format.startswith('binary'):
             print(r'DataFormat=BINARY', file=fout)
 
         if multiplexed:
-            print(r'; DataOrientation: MULTIPLEXED=ch1,pt1, ch2,pt1 ...', file=fout)  # noqa: E501
+            print(r'; Data orientation: MULTIPLEXED=ch1,pt1, ch2,pt1 ...', file=fout)  # noqa: E501
             print(r'DataOrientation=MULTIPLEXED', file=fout)
 
         print(r'NumberOfChannels={}'.format(len(data)), file=fout)
         print(r'; Sampling interval in microseconds', file=fout)
         print(r'SamplingInterval={}'.format(int(1e6 / sfreq)), file=fout)
         print(r'', file=fout)
 
-        if fmt.startswith('binary'):
+        if format.startswith('binary'):
             print(r'[Binary Infos]', file=fout)
             print(r'BinaryFormat={}'.format(bvfmt), file=fout)
             print(r'', file=fout)
 
         print(r'[Channel Infos]', file=fout)
         print(r'; Each entry: Ch<Channel number>=<Name>,<Reference channel name>,', file=fout)  # noqa: E501
-        print(r'; <Resolution in "unit">,<unit>,Future extensions…', file=fout)
+        print(r'; <Resolution in "Unit">,<Unit>, Future extensions..', file=fout)  # noqa: E501
```

Comment:
again, copying how it is in "real" BV files --> including the oddly two (!) trailing dots